lib/TPP/DefaultTppPasses.cpp

```diff
@@ -246,7 +246,6 @@ struct TppConversionPass : public TppConversionBase<TppConversionPass>,
 
     // Convert all higher level dialects to TPP.
     pm.addPass(createConvertLinalgToTppPass());
-    pm.addPass(createCombineTppPass());
```

Comment:
Please do not remove this pass from the pipeline in this PR. Let's do that once your PR is merged, as a clean up, while removing the TPP dialect.

Reply:
reverted

Reply:
Still seems to be here in the last diff